### DISCUSSION TITLE
Welcome to Learning-Management-System Discussions!

### DISCUSSION THREAD

**Discussion by neo1427** [neo1427/Learning-Management-System#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by neo1427** [comment]
**Guys I hope you have gone through the assigned tasks.**

- Reference file also had been added for the issues
- Soon Umls will be added.

**Reply by Vamsineo22** [reply]
yeah, working on it.

**Comment by neo1427** [comment]
Well done Srijan for create the base part of the project

**Comment by neo1427** [comment]
@srijanNeo can added the validations to each class in the model folder

**Reply by srijanNeo** [reply]
I have added the validations required for every field on top of the entity class as requested, that should be implemented with field names and if there are any doubts reach out to me.
